examples/screen-publications.json

```diff
@@ -69,588 +157,94 @@
-                            "id": "096ec6fc-2722-4907-82ef-8694b3f38240",
-                            "name": "Suisse Garantie",
-                            "image": {
-                                "cdn_url": "https://denner.imgix.net/eco-labels/096ec6fc-2722-4907-82ef-8694b3f38240/suisse-garantie.png"
-                            }
-                        },
-                        {
-                            "id": "920795b5-3dbf-4a56-875b-76837cd4eca3",
-                            "name": "UTZ",
-                            "image": {
-                                "cdn_url": "https://denner.imgix.net/eco-labels/920795b5-3dbf-4a56-875b-76837cd4eca3/utz.png"
-                            }
-                        }
-                    ],
-                    "packshot": {
-                        "id": "5054223d-3830-4abb-a2aa-4de55250d6a4",
-                        "url": "https://denner-assets.s3-eu-west-1.amazonaws.com/assets/5054223d-3830-4abb-a2aa-4de55250d6a4/091121.png",
-                        "cdn_url": "https://denner.imgix.net/assets/5054223d-3830-4abb-a2aa-4de55250d6a4/091121.png",
-                        "quantity": 1,
-                        "created_on": "2016-01-07T15:42:19+02:00"
-                    },
-                    "supplier": {
-                        "code": 36323
-                    },
-                    "country": {
-                        "id": "AR",
-                        "title": [
-                            {
-                                "locale": "de",
-                                "value": "Argentinien"
-                            },
-                            {
-                                "locale": "fr",
-                                "value": "l'Argentine"
-                            },
-                            {
-                                "locale": "it",
-                                "value": "Argentina"
-                            }
-                        ],
-                        "flag": {
-                            "image_url": "https://denner-assets.s3-eu-west-1.amazonaws.com/flags/AR.png"
-                        }
-                    }
-                },
+            "footer": []
+        },
+        {
+            "id": "ad4a6d47-2506-41a7-9cdc-b9f81bc5b56c",
+            "name": "Argento Selección Malbec (KW 29/2020) für Werbescreens",
+            "year": 2020,
+            "week": 29,
+            "sort_index": 325,
+            "created_on": "2020-07-01T08:41:27+0200",
+            "validity": {
+                "from": "2020-07-14",
+                "to": "2020-07-20",
+                "text": []
+            },
+            "promotion_type": {
+                "id": "089f8537-981a-42ba-9b0a-b3cba4a1c7d5",
+                "code": "FLUGBLATT",
+                "name": "Dennerwoche"
+            },
+            "automatic": true,
+            "article_count": 1,
+            "screen": {
+                "id": "cache-register",
```

Comment:
Why not `pos` (it's the acronym for Point of Sales)?

Reply:
ok